scripts/vcf_sample_filter.py

```diff
@@ -34,6 +34,6 @@
     sf = SampleFilter(infile=args.file, outfile=args.o,
                       filters=args.f, invert=args.invert)
     if args.f is None:
-        print "Samples:"
+        print("Samples:")
```

Comment:
I'd use `PRINT` from `dcicutils.misc_utils` here to highlight that the print is part of the action of the script.